src/gerbil/gxc-main.ss

```diff
@@ -127,7 +130,16 @@
               (else
                `("-e" ,include-gambit-sharp)))))
           (lp rest))
-         (("-prelude")
+         (("-target")
+          (match rest
+            ([opt . rest]
+             (add-gsc-option! ["-target" opt])
```

Comment:
this is a mistake, pass the target to the driver and let it do it.

Reply:
You mean c/p add-gsc-option! over to compiler/driver.ss and then do that addition anywhere gsc is called? Or beforehand, like it's here but move it? To where?

Reply:
Ah, the make doesn't call gxc, it does so manually, and/or passes the keyword to something? Then copy/pasta indeed. Will do.

Reply:
Done?